src/interfaces/todo-item-interface.ts

```diff
@@ -23,8 +22,8 @@ export interface TodoItemIF {
   paymentToken?: number
   payErrorObj?: PaymentErrorIF
 
-  // alterations and corrections only
-  comments?: Array<any>
```

Comment:
No todo items have the detail comments array.

But correction items do have the correction comment property. >>>